testing.rst

```diff
@@ -815,6 +815,92 @@ You can remove an existing field, e.g. a tag::
 .. index::
    pair: Tests; Configuration
 
+Testing file upload using CollectionType
+........................................
+
+By default, a CollectionType isn't available in the form during functional tests 
+if no data are passed by default, sometimes, you need to upload multiple files (using this CollectionType), 
+keeping in mind this logic, it can be hard to handle the upload.
+
+Let's imagine a simple form which define a collection of FileType::
+
+    <?php
+
+    namespace App\Form;
+
+    use App\Entity\Entity;
+    use Symfony\Component\Form\AbstractType;
+    use Symfony\Component\Form\FormBuilderInterface;
+    use Symfony\Component\OptionsResolver\OptionsResolver;
+    use Symfony\Component\Form\Extension\Core\Type\CollectionType;
+    use Symfony\Component\Form\Extension\Core\Type\FileType;
+
+    class DefaultUploadType extends AbstractType
+    {
+        /**
+         * {@inheritdoc}
+         */
+        public function buildForm(FormBuilderInterface $builder, array $options)
+        {
+            $builder
+                ->add('files', CollectionType::class, [
```

Comment:
You opened array, but did not close it. And also I notices that usually in documentation is used `array()` definition and not `[]`. Please check it, to be consistent with whole documentation.

Reply:
We now use the short array syntax in Symfony core and docs from 3.4